client/payment-details/timeline/map-events.js

```diff
@@ -267,34 +267,43 @@ const feeBreakdown = ( event ) => {
 		fee_rates: { history },
 	} = event;
 
-	const feeLabelMapping = ( fixedRate ) => ( {
-		base:
-			0 !== fixedRate
-				? /* translators: %1$s% is the fee amount and %2$s is the fixed rate */
-				  __( 'Base fee: %1$s%% + %2$s', 'woocommerce-payments' )
-				: /* translators: %1$s% is the fee amount */
-				  __( 'Base fee: %1$s%%', 'woocommerce-payments' ),
+	const feeLabelMapping = ( fixedRate, capped ) => ( {
+		base: ( () => {
```

Comment:
I tested the PR with a UK account (which has an EUR deposit currency) and a DE card. In this case there are no FX or international card fees, meaning that only the base fee remains. I was wondering why I was not seeing the change, until I noticed that there are two conditions at the beginning of `feeBreakdown`:

- One of them checks whether there is any fee history within the event.
- The second one hides fee history whenever there is a single fee in the breakdown (base fee only), which was implemented [in this PR](https://github.com/Automattic/woocommerce-payments/pull/2928).

In both cases the breakdown is not displayed, leading to only the output of `composeFeeString`, which did not contain the "capped" text.

I wrote the required code in [this branch](https://github.com/Automattic/woocommerce-payments/compare/add/display-fee-cap-breakdown...add/display-fee-cap-breakdown-base-only), but I'm not sure whether `composeFeeString` is the place to put it. If you agree with this approach, you can just use the commit from that branch and apply it on top of this one. If not, feel free to implement something similar.